website/client/src/assets/scss/button.scss

```diff
@@ -32,56 +34,59 @@
     opacity: 0.75;
     box-shadow: 0 1px 3px 0 rgba(26, 24, 29, 0.12), 0 1px 2px 0 rgba(26, 24, 29, 0.24);
     background-color: $gray-700;
-    border: 1px solid transparent;
+    border: 2px solid transparent;
 
     .svg {
       color: $gray-300;
     }
   }
 
   &.with-icon {
-    height: 2rem; // otherwise would something set the height to 33px
+    height: 32px; // otherwise would something set the height to 33px
```

Comment:
What sets the height to 33px?

Reply:
That was a pre-existing comment; I don't know why and thought it would be best to leave it as-is.